.eslintrc.js

```diff
@@ -1,10 +0,0 @@
-module.exports = {
-  root: true,
-  // This tells ESLint to load the config from the package `eslint-config-custom`
-  extends: ['custom'],
-  settings: {
-    next: {
-      rootDir: ['apps/*/'],
-    },
-  },
-};
```

Comment:
why this lines are removed..?

Reply:
it conflicts in my editor, as a result formatting and saving any changes tooks too much time that's why I temporarily removed those